parts/k8s/kuberneteswindowsfunctions.ps1

```diff
@@ -17,23 +17,36 @@ function DownloadFileOverHttp
         [Parameter(Mandatory=$true)][string]
         $DestinationPath
     )
-    $secureProtocols = @()
-    $insecureProtocols = @([System.Net.SecurityProtocolType]::SystemDefault, [System.Net.SecurityProtocolType]::Ssl3)
 
-    foreach ($protocol in [System.Enum]::GetValues([System.Net.SecurityProtocolType]))
+    # First check to see if a file with the same name is already cached on the VHD
+    $fileName = [IO.Path]::GetFileName($Url)
+    $search = [IO.Directory]::GetFiles($global:CacheDir, $fileName, [IO.SearchOption]::AllDirectories)
+
+    if ($search.Count -ne 0)
+    {
+        Write-Log "Using chaced version of $fileName - Copying file from $($search[0]) to $DestinationPath"
+        Move-Item -Path $search[0] -Destination $DestinationPath -Force
+    }
+    else 
     {
-        if ($insecureProtocols -notcontains $protocol)
+        $secureProtocols = @()

```

Comment:
updated

Reply:
that's a good question. I thought that GitHub downloads needed this, but maybe it's no longer needed. Since the caching is passing without it - maybe we can remove it and use the normal security protocol policy?